### PR TITLE
fix: flaws in web/accessibility

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/web/accessibility/accessibility_colon__what_users_can_to_to_browse_safely/index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@ <h3 id="Use_modern_browsers._Learn_personalization_and_accessibility_settings">U
 
 <p>Chrome 74 added support for <strong><code>reduced-motion-query</code></strong>, it has even added support to detect whether the user has requested a reduced motion experience. The user would access this through an accessibility interface, as seen below.</p>
 
-<p><img alt="Screen shot showing how to turn off animations in Windows10" src="https://mdn.mozillademos.org/files/16662/android-remove-animations.png" style="height: 493px; width: 472px;"></p>
+<p><img alt="Screen shot showing how to turn off animations in Windows10" src="android-remove-animations.png"></p>
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/accessibility_colon__what_users_can_to_to_browse_safely/index.html:36]:
```suggestion
<p><img alt="Screen shot showing how to turn off animations in Windows 10." src="android-remove-animations.png"></p>
```

Little tweaks for better pronunciation.

File: files/en-us/web/accessibility/implementing_msaa_server/index.html [hunk 2]
```diff
@@ -459,7 +459,7 @@ <h3 id="The_Accessible_Tree_vs._the_DOM_Tree_2"><a id="The_Accessible_Tree_vs._t
 
 <h3 id="MSAA_tree_vs._DOM_tree_-_what's_the_relationship">MSAA tree vs. DOM tree - what's the relationship?</h3>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/6227/tree-relativity.gif" style="border-style: solid; border-width: 1px; height: 506px; width: 696px;"></p>
+<p><img alt="" src="tree-relativity.gif" style="border-style: solid; border-width: 1px;"></p>
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/implementing_msaa_server/index.html:462]: Is there a reason this image is nulled? It seems like it contains important info.

File: files/en-us/web/accessibility/understanding_colors_and_luminance/index.html [hunk 3]
```diff
@@ -58,7 +58,7 @@ <h3 id="Conversions">Conversions</h3>
 
 <p>Conversions are not casual.   See the Stack Exchange article,  "<strong><a href="https://stackoverflow.com/questions/596216/formula-to-determine-brightness-of-rgb-color">Formula to Determine the Brightness of RGB Color</a></strong>".   There are many online tools that can convert RGB to HSL for you, with both the RGB hexadecimal and CSS function syntax.   A great example of a tool that converts colors for you is Tom Jewett's "<strong><a href="http://colortutorial.design/miniColors.html">mini color selector</a></strong>" with <strong>HSL, RGB, Hex</strong> options for checking contrast in the browser. </p>
 
-<p><img alt="Functional in Firefox, Chrome, and Opera browsers" src="https://mdn.mozillademos.org/files/16756/TomJewettMiniColorPicker.PNG" style="height: 255px; width: 520px;"></p>
+<p><img alt="Functional in Firefox, Chrome, and Opera browsers" src="tomjewettminicolorpicker.png"></p>
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/understanding_colors_and_luminance/index.html:61]:
```suggestion
<p><img alt="Functional in Firefox, Chrome, and Opera browsers." src="tomjewettminicolorpicker.png"></p>
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]

File: files/en-us/web/accessibility/implementing_msaa_server/index.html (hunk 2 above)
Comment (nschonni) [reply files/en-us/web/accessibility/implementing_msaa_server/index.html:462]: No idea, there are a lot of empty alt text values in the repo that should probably be looked at. The automated fixups just pull down the external images.

Comment (Ryuno-Ki) [reply files/en-us/web/accessibility/implementing_msaa_server/index.html:462]: Could we have an issue to remind us to check the alt attributes?

**Review by junauseng — APPROVED** [review]

**Review by ericwbailey — APPROVED** [review]

**Pull request merged**